books/RayTracingInOneWeekend.html

```diff
@@ -920,15 +920,15 @@
     class hittable_list: public hittable {
         public:
             hittable_list() {}
-            hittable_list(hittable* object) { add(object); }
+            hittable_list(shared_ptr<hittable> object) { add(object); }
```

Comment:
This caught my eye.
It's a little strange to have a list class that has a valued-constructor of the first element.

But going through the usage in the code base, it's pretty reasonable.
It's only used in places where the list only has a single value.